### PR TITLE
sort id_set.json in order to avoid conflicts

### PR DESCRIPTION
**Pull request by anara123**
## Status
Ready

## Description
- sorted the id_set.json in order to avoid conflicts that will happen when there are changes in master
- sort the id_set.json every time update_id_set.py running - in order to make sure that the id_set.json stays sorted, even if someone changed id_set.json manually

## Does it break backward compatibility?
   - No

### CODE REVIEW

File: Tests/scripts/update_id_set.py [hunk 1]
```diff
@@ -53,6 +54,18 @@ def run_git_command(command):
     return output
 
 
+def insert(seq, keys, item, keyfunc=lambda v: v):
```
Comment (glicht) [inline Tests/scripts/update_id_set.py]: seems that this function is not used. I think we can remove it.

File: Tests/scripts/update_id_set.py [hunk 2]
```diff
@@ -503,10 +521,12 @@ def update_id_set():
 
     if added_files or modified_files:
         new_ids_dict = OrderedDict()
-        new_ids_dict['scripts'] = script_set
-        new_ids_dict['playbooks'] = playbook_set
-        new_ids_dict['integrations'] = integration_set
-        new_ids_dict['TestPlaybooks'] = test_playbook_set
+        # we sort each time the whole set in case someone manually changed something
+        # it shouldn't take too much time
+        new_ids_dict['scripts'] = sort(script_set)
```
Comment (ronykoz) [inline Tests/scripts/update_id_set.py]: These sorts should take a lot, no? you are sorting a file of 15K lines..

Comment (anara123) [reply Tests/scripts/update_id_set.py]: it actually works very fast because we have it all in the memory and we sort in total ~1000 items in array

Comment (anara123) [reply Tests/scripts/update_id_set.py]: milliseconds

**Pull request merged**